src/helpers/uniqueAssetTraitDisplayTypeCompareFunction.ts

```diff
@@ -0,0 +1,33 @@
+import { UniqueAsset } from '@rainbow-me/entities';
+
+type Trait = UniqueAsset['traits'][number];
+type ReturnType = -1 | 0 | 1;
+
+// lower number = higher precedence
+const displayTypeRanks = new Map<string | undefined | null, number>([
```

Comment:
SHould it be somewhere else as a constant? Do ou use certain constants formatting like `UPPPER_CASE`?